README.md

```diff
@@ -6,39 +6,38 @@
 
-Arktos aims to be an open source solution to address key challenges of large scale clouds, including system scalability, resource efficiency, multitenancy, cross-AZ resiliency, and the native support for the fast-growing modern workloads such as containers and serverless functions. 
-
-----
-
-## Key Features of Arktos
+Arktos aims to be an open source solution to address key challenges of large-scale clouds, including system scalability, resource efficiency, multitenancy, edge computing, and the native support for the fast-growing modern workloads such as containers and serverless functions. 
 
+## Architecture
+![](docs/design-proposals/arch/project_architecture.png)
+## Key Features
 
 ### Large Scalability
 
-Arktos achieves a scalable architecture by partitioning and replicating system components, including API Server, storage, controllers and data plane. The eventual goal of Arktos is to support 100K nodes with a single cross-AZ control plane.
+Arktos achieves a scalable architecture by partitioning and scaling out components, including API Server, storage, controllers and data plane. The eventual goal of Arktos is to support 300K nodes with a single regional control plane.
 
 ### Multitenancy
 
-Arktos implements a hard multi-tenancy model to meet the strict isolation requirement highly desired by a typical public cloud environment. Tenant is a built-in object in the system, and some flexible multi-tenancy models can also be supported with customized resource authorization rules.
-
-### Native VM Support
+Arktos implements a hard multitenancy model to meet the strict isolation requirement highly desired by public cloud environment. It's based on the virtual cluster idea and all isolations are transparent to tenants. Each tenant feels it's a dedicated cluster for them. 
 
-In addition to container orchestration, Arktos implements a built-in support for VMs. In Arktos a pod can contain either containers or a VM. They are scheduled the same way and launched by node agent using different runtime servers. VMs and containers are both the first-class citizens in Arktos.
+### Unified Container/VM Orchestration
 
+In addition to container orchestration, Arktos implements a built-in support for VMs. In Arktos a pod can contain either containers or a VM. They are scheduled the same way in a same resource pool. This enables cloud providers use a single converged stack to manage all cloud hosts.
 
-### More Planned Features
+### More Features
 
-More features are planned but not started yet, including intelligent scheduling, in-place resource update, QoS enforcement, etc.
+There are more features under development, such as cloud-edge scheduling, in-place vertical scaling, etc. Check out [the project introduction] (https://docs.google.com/presentation/d/1PG1m27MYRh4kuq654W9HvdoZ5QDX9tWxoCMCfeOZUrE/edit#slide=id.g8a27d34398_8_0) for more information.
```

Comment:
Link formatting issue, please take a look.

Reply:
Fixed. Thank you for catching this.  My local editor (MacDown) tolerate this extra space so I didn't find it locally.